rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -3275,6 +3275,8 @@ void main_window::CreateConnects()
 	connect(ui->welcomeAct, &QAction::triggered, this, [this]()
 	{
 		welcome_dialog* welcome = new welcome_dialog(m_gui_settings, true, this);
+
+		// welcome dialog is auto-flagged (in its constructor) as WA_DeleteOnClose, so its pointer is no more usable after welcome->open()
```

Comment:
This is just plain wrong and it makes no sense even mentioning this here

Reply:
Yes, the comment should be more articulated for the `open()` method. It was provided more to highlight the fact that there was no need to handle the pointer deletion, but that is common to other dialogs opened in this source file. So simply removing it